app/batching/publish-batch.js

```diff
@@ -1,7 +1,15 @@
-const publishBatch = async (batch, filename, content) => {
-  console.info(`Publishing ${batch.scheme.name} ${batch.ledger} batch ${batch.sequence}`)
-  console.info(`Filename: ${filename}`)
-  console.info(`content: ${JSON.stringify(content)}`)
+const { getOutboundBlobClient } = require('../storage')
+const { writeToString } = require('@fast-csv/format')
+
+const publishBatch = async (filename, content) => {
+  const body = await stringifyContent(content)
+  const outboundBlobClient = await getOutboundBlobClient(filename)
+  await outboundBlobClient.upload(body, body.length)
+  console.info(`Published ${filename}`)
+}
+
+const stringifyContent = async (content) => {
```

Comment:
> All looks good but just wanted to check whether you need the archive and quarantine config before approving.

We do need them but I'm not sure whether the creation of them should be the responsibility of this repo or not.  Once the file is created then something (Logic App, Function etc) will take these files and move them to DAX.  If that process succeeds they need to be archived.  If it fails they need to be quarantined.

My first thought was that second step isn't a concern of this service so folders shouldn't be coupled to it.  However, I changed my mind after thinking about the naming convention of the folders.  Naming them aligned with the service means it could get difficult to understand what each folder is if they're not consistent and although split across logic app(?) and k8s, they're all the same submit to DAX stage.

But now you've highlighted it, I starting to convince myself that they shouldn't be included here as they're the dependencies of a separate service and I'll I've done is interlink those dependencies.

I'm going to remove them and revisit the naming convention of the folders as I think that is what sent me down this wrong path.

Reply:
> Is this a bit redundant? Can you just call `writeToString(content)` above on line 5?

Yes it is.  Originally I used a different approach to writing the CSV that was a bit more involved in converting the array of vendor and ledger lines into a CSV string.  I felt that way was too overcomplicated so went for a simpler library to do it.  

Didn't appreciate that by doing that it made that function redundant.  Now removed.